### ISSUE TITLE
Create new FMB R&S page: Regional loan center

### ISSUE THREAD

**Issue by aprocik1** [department-of-veterans-affairs/va.gov-team#92442]

**Comment by aprocik1** [comment]
@d-beyer, you're right that we need to design the locations lists differently on this page. For example, I don't think we should bullet points for a list of one ("a state listed here:").

Some suggested edits:
- Remove the bullet point and "a state listed here:" content when the location is just for states
- Remove the "a state listed here:" content when there's only one state
- Order bullet points in alphabetical order, but always lead with states
- Add "the" in front of DC and Virgin Islands to match the GI Bill page

Check out the draft in Drupal: https://preview-prod.vfs.va.gov/preview?nodeId=73554

Any and all feedback welcome!

**Comment by d-beyer** [comment]
@bethpotts -- The regional loan center address R&S page is ready for your review. Let us know if you have any questions or feedback.

[Preview: Regional loan center addresses for VA home loan applications](http://preview-prod.vfs.va.gov/preview?nodeId=73554)

**Comment by aprocik1** [comment]
Update: Beth and I reviewed this page together today. We're going to remove the bullet points from this page entirely--each section will have a list of locations in body copy.

I'll make this update tomorrow. Then, I'll email this page to David Ahn to review. In that email, I'll include information about the FMB migration and offer an FYI about the other changes (moving and redirecting the home loans for survivors page).

**Comment by aprocik1** [comment]
Update: I follow up with David Ahn on 10/21. Then he was OOO for about two weeks. I'll align on a follow up with Beth today.

**Comment by aprocik1** [comment]
Update: Beth committed to following up with David Ahn to get this page reviewed and approved in January. After we publish this page, we'll add a crosslink to it in the "more information and resources" section of the [FMB housing sub hub](https://preview-prod.vfs.va.gov/preview?nodeId=65665) page (Drupal preview).

**Comment by aprocik1** [comment]
Request for Triana's support on this page: Beth and Danielle want us to explore using regions in the headers, rather than offices. For example:

> ## Northeast
> If you live in on these northeastern locations, send your application to the Cleveland Regional Loan Center: New Hampshire, Vermont, etc.
> 
> This is the address for the Cleveland Regional Loan Center:
> 
> > Department of Veterans Affairs 
> > Cleveland Regional Loan Center 
> > 1240 East Ninth Street 
> > Cleveland, OH 44199

If a region has more than one loan center, we may need to add H3s for further specificity. If a loan center spans more than one region, we may need to provide that loan center address in multiple H2s. We should also consider if we can avoid naming the loan center ("Cleveland Regional Loan Center") in body copy. Note that the loan center name should remain unchanged in the address block.

The goal of these updates to make this page easier to navigate, similar to the simple table on this page: [Hearing coordinators for the Board of Veterans’ Appeals](https://www.va.gov/resources/hearing-coordinators-for-the-board-of-veterans-appeals/)

<details>
<summary>Screenshot of draft page in Drupal</summary>

![Image](https://github.com/user-attachments/assets/32c77e13-01a5-4a7b-a6f3-7991750e3829)

</details>

<details>
<summary>References for possible region names</summary>

- https://education.nationalgeographic.org/resource/united-states-regions/
- https://www2.census.gov/geo/pdfs/maps-data/maps/reference/us_regdiv.pdf

</details>

P.S. Our thinking on this page should also inform future updates to this page: [Regional processing office addresses for GI Bill applications](https://www.va.gov/resources/regional-processing-office-addresses-for-gi-bill-applications/)

**Comment by aprocik1** [comment]
Current status: This page is ready for SME review, pending Beth's approval:

[Regional loan center addresses for VA home loan applications](http://preview-prod.vfs.va.gov/preview?nodeId=73554) (Drupal)

One note: The jumplinks are formatted using HTML code and the published URL for this page. You won't be able to test them until after the page is published. If you need to update the jumplinks, I would recommend uploading a new CSV in Drupal. Here's the SharePoint document for downloading the CSV (reference the "final" tab):

[2025 Regional loan center page](https://dvagov.sharepoint.com/:x:/s/SitewideCAIA/EUfyGlx6KIFGuTcZ0bfOgCcB6ZDjb3x5_wobhmZuF8Wc0A?e=cNzewS) (SharePoint)

And P.S.: Here's the document we used to draft this page, which includes the previous page design without the table:

[2025 Regional loan center addresses for VA home loan applications](https://dvagov.sharepoint.com/:w:/s/SitewideCAIA/ERxuxDB8iglGlaCzYRJZ7EkBSg_blFjldOGeYpxzkbADFA?e=fiQohe) (SharePoint)

**Comment by RLHecht** [comment]
I sent this to David for an accuracy check on 6/17.
